### ISSUE TITLE
Contact not added to SendInBlue list

### ISSUE THREAD

**Issue by russback** [verbb/formie#403]
I am seeing a contact added to SendInBlue and the field mappings are working as expected. However the contact is not added to the list selected.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.23
- Craft version: 3.6.10
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.24",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.25](https://github.com/verbb/formie/releases/tag/1.3.25)
